docs/Reference/Pantheon-API-Methods.md

```diff
@@ -1344,6 +1660,25 @@ Object - [Transaction object](Pantheon-API-Objects.md#transaction-object), or `n
     }
     ```
 
+    ```bash tab="curl GraphQL request"
+    curl -X POST -H "Content-Type: application/json" --data '{"query": "{block(number:\"0x4f4f\") {transactionAt(index: 0) {block{hash} hash}}}"}' http://localhost:8547/graphql
```

Comment:
Number should not be a hex string:
```suggestion
   curl -X POST -H "Content-Type: application/json" --data '{"query": "{block(number:82990) {transactionAt(index: 0) {block{hash} hash}}}"}' http://localhost:8547/graphql
```
Result will need to be recalculated to be the same block number as JSON-RPC

Reply:
Changed to a number but haven't addressed the wider issue of being the same block number as JSON-RPC at this point.  I've added this to the wider point to updating all requests to be against a public testnet and adding an overview about using Infura to access the endpoint.  There's not GraphQL endpoints available on Infura yet so need to wait for these to be addd.

Reply:
Have added that this specific method example needs updated to https://pegasys1.atlassian.net/browse/PAN-2714